feature/platform/tests/telemetry_inventory_test/metadata.textproto

```diff
@@ -38,7 +41,9 @@ platform_exceptions: {
   }
   deviations: {
     backplane_facing_capacity_unsupported: true
+    fan_tray_type_unsupported: true
     model_name_unsupported: true
+    switch_chip_id_unsupported: true
```

Comment:
switch chip id is supported AFAIK.

Reply:
Oh right, thanks! Didn't mean to add that here:)